nni/compression/tensorflow/compressor.py

```diff
@@ -167,6 +211,43 @@ def compress(self):
         self._update_mask()
         return self.compressed_model
 
+    def export_model(self, model_path, mask_path=None):
+        """
+        Export pruned model and optionally mask tensors.
+
+        Parameters
+        ----------
+        model_path : path-like
+            The path passed to ``Model.save()``.
+            You can use ".h5" extension name to export HDF5 format.
+        mask_path : path-like or None
+            Export masks to the path when set.
+            Because Keras cannot save tensors without a ``Model``,
+            this will create a model, set all masks as its weights, and then save that model.
+            Masks in saved model will be named by corresponding layer name in compressed model.
```

Comment:
so even the mask file has different format between pytorch and tensorflow?...

Reply:
PyTorch masks are exported as PyTorch tensors. So there's no way to make TF masks compatible.